community/iptables/sources

```diff
@@ -0,0 +1 @@
+http://www.netfilter.org/projects/iptables/files/iptables-1.8.4.tar.bz2
```

Comment:
Change this to: `https://www.netfilter.org/projects/iptables/files/iptables-1.8.4.tar.bz2` (https)

Reply:
nice